amundsen_application/api/__init__.py

```diff
@@ -22,5 +22,5 @@ def healthcheck() -> Tuple[str, int]:
 
 def favicon() -> IO[bytes]:
     """ TODO: Design team should provide us with a default icon """
-    return send_from_directory(os.path.join(app.root_path, 'static/images'),
-                               'favicon.ico', mimetype='image/vnd.microsoft.icon')  # pragma: no cover
+    return send_from_directory(os.path.join(app.root_path, 'static/images/favicon.png'),
+                               mimetype='image/vnd.microsoft.icon')  # pragma: no cover
```

Comment:
I believe `send_from_directory` will return an error as the `directory` and `filename` parameters are required. That being said, this function related API are not used. 

The favicon is linked on line 14 of `static/templates/index.html`.
```
<link rel="shortcut icon" href="/static/images/favicon.png">
```

I would recommend deleting this API and function.

Reply:
@thenaturalist  @danwom I help to resolve some merge conflict and update the pr per Daniel's suggestion. Could you both take a look?

Reply:
You should also delete the route added on line 9.

Looks like whoever (me) added the favicon forgot to do some cleanup. 😄